tests/e2e/tests/070_deprecate/010_unauth_deprecate.ts

```diff
@@ -0,0 +1,18 @@
+import { npmPublish, npmLogin, executeSilent, deprecate } from '../../utils/process';
+import { createPackageJson } from '../../utils/utils';
+
+export default function() {
+  return Promise.resolve()
+    .then(() => createPackageJson('package.json', 'test-package', '0.0.1'))
+    .then(() => npmLogin('admin', 'blabla', 'foo@bar.com'))
+    .then(() => npmPublish())
+    .then(() => executeSilent('npm logout'))
+    .then(() => deprecate('test-package', 'deprecate message'))
+    .then(res => {
```

Comment:
can you add `npm install test-package` after that command and check if returned deprecate message is same as you setted it before (deprecate message)?